.babelrc

```diff
@@ -1,4 +1,8 @@
 {
-  "presets": ["es2015", "react"],
+  "presets": ["react", ["env", {
+    "targets": {
```

Comment:
Should this mirror what we use for autoprefixer: https://github.com/cerner/terra-core/blob/master/packages/terra-site/postcss.config.js#L12-L16

Reply:
Potentially. Its slightly different because whereas CSS uses prefixes  JavaScript its either ther e or it isn't. That means that being we support IE10 its a pretty strong bet that will include anything chrome, safari and firefox is missing that we do support.   Once we no longer support IE10, then we'll probably want to be more nuanced about it. Because supporting IE10 basically means everythign has to be transpiled.

Reply:
Lets update this to `"browsers": ["last 2 versions", "iOS >= 10"]` 
IE 10 & 11 will be captured by the `last 2 versions` string.

This config matches the same results as our current autoprefixer setup. 

<img width="1440" alt="screen shot 2017-10-09 at 9 49 04 am" src="https://user-images.githubusercontent.com/633148/31344235-b9c761a0-acd7-11e7-942a-a71d9b6f3bdc.png">

We should also update autoprefixer to use the same config.

If we drop the `iOS >= 10` part and go with `"browsers": ["last 2 versions"]`, we'd drop iOS 10.0-10.2 and iOS 10 "support" would start at `10.3`. I'd defer to @mjhenkes on what level of iOS 10 support we need.

<img width="1440" alt="screen shot 2017-10-09 at 9 49 21 am" src="https://user-images.githubusercontent.com/633148/31344333-fc3ac4be-acd7-11e7-8ae7-82cfb0b63c0e.png">

Reply:
We currently support iOS 10.2 and up.

Reply:
Ok sounds like we need `"browsers": ["last 2 versions", "iOS >= 10"]`